feature/main/src/main/java/io/github/droidkaigi/confsched2023/main/MainScreen.kt

```diff
@@ -63,22 +78,36 @@ interface MainNestedGraphStateHolder {
     fun onTabSelected(mainNestedNavController: NavController, tab: MainScreenTab)
 }
 
+enum class NavigationType {
+    BOTTOM_NAVIGATION, NAVIGATION_RAIL
+}
+
 @Composable
 fun MainScreen(
+    windowSize: WindowSizeClass,
+    displayFeatures: ImmutableList<DisplayFeature>,
```

Comment:
What do you think you are going to use this? 👀

Reply:
1) windowSize: to calculate WindowWidthSizeClass and apply proper navigationType(in this pr) and content pane type(in next PR, apply dual-pane(as known as List-Detail Pattern) or single-pane)

2) displayFeatures: When WindowWidthSizeClass is Medium, content arrangement type should be determined by device folding posture. (according to material 3 sample application [reply](https://github.com/android/compose-samples/blob/main/Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt#L99))

Reply:
As [explained](https://github.com/DroidKaigi/conference-app-2023/issues/460#issuecomment-1684697989), list-detail pattern will apply in this screen. So `displayFeatures` will be used to implement it.

Reply:
Great! Thanks